.github/workflows/test-code.yml

```diff
@@ -30,22 +30,22 @@ jobs:
           source-url: https://nuget.pkg.github.com/graphql-dotnet/index.json
         env:
           NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      - name: Install dependencies
+        working-directory: src
+        run: dotnet restore
       - name: Check formatting
         if: ${{ startsWith(matrix.os, 'ubuntu') }}
         working-directory: src
         run: |
-          dotnet tool install -g dotnet-format
-          dotnet format --check -v diag || (echo "Run 'dotnet format' to fix formatting issues" && exit 1)
-      - name: Install dependencies
```

Comment:
Some analyzers require already installed dependencies, so this step has been moved up.